src/webgpu/format_info.ts

```diff
@@ -1236,7 +1236,7 @@ export function filterFormatsByFeature<T>(
 }
 
 export function isCompressedTextureFormat(format: GPUTextureFormat) {
-  return format in kCompressedTextureFormats;
+  return (kCompressedTextureFormats as GPUTextureFormat[]).includes(format);
```

Comment:
Ah... oops, I misread the code as `in kCompressedTextureFormatInfo`. Would that work?

Reply:
yea, that's better